frontend/src/framework/theme/styles/colors.styles.ts

```diff
@@ -7,13 +7,10 @@ const colors = {
         50: '#e2e1ec',
     },
     brand: {
-        900: '#1a365d',
-        200: '#b3e0ff',
-        secondary: {
-            300: '#ff6e1c',
-            600: '#eb5500',
-            900: '#e13b00',
-        },
+        orange: '#ff6e1c',
+        hover: '#eb5500',
+        green: '#3BAF28',
+        error: '#e13b00',
```

Comment:
Maybe you should keep secondary prop in brand and just simply add green property to brand? Because some of the PRs(like FE signup and signin) are using the secondary property. What do you think?

Reply:
It's difficult when everyone has created their own style block, but I'll make the change.